### DISCUSSION TITLE
diffrent port

### DISCUSSION THREAD

**Discussion by smg4vm-owner** [rejetto/hfs#325] [Q&A] [answered]
Hello, is there a way to use a different port using the command line, port 80 can't be used as a web server is already being hosted there.

**Comment by rejetto** [comment, marked as answer]
Sure, it's --port 80
Complete instructions on all commands is a available at https://github.com/rejetto/hfs#configuration

**Reply by smg4vm-owner** [reply]
So, "hfs.exe --port (port here)"?

**Reply by rejetto** [reply]
Yes

**Reply by smg4vm-owner** [reply]
Hmm, still can't access it.

**Reply by smg4vm-owner** [reply]
Says that Firefox can't access localhost:(my port number).

**Reply by smg4vm-owner** [reply]
Never mind, just took a while for the server to fully load, thanks!

**Comment by rejetto** [comment]
Btw, When you start HFS and port 80 is busy, it's already suggesting how to pass another port at command line
